secondarylib/src/main/scala/com/socrata/datacoordinator/secondary/SecondaryWatcher.scala

```diff
@@ -65,9 +73,12 @@ class SecondaryWatcher[CT, CV](universe: => Managed[SecondaryWatcher.UniverseTyp
         log.info(">> Syncing {} into {}", job.datasetId, secondary.storeId)
         if(job.replayNum > 0) log.info("Replay #{} of {}", job.replayNum, maxReplays)
         if(job.retryNum > 0) log.info("Retry #{} of {}", job.retryNum, maxRetries)
+        var noCatch = false
+        val startingMillis = System.currentTimeMillis()
         try {
           playbackToSecondary(secondary, job)
           log.info("<< Sync done for {} into {}", job.datasetId, secondary.storeId)
+          noCatch = true
```

Comment:
create a function def releaseClaimAndLogMessage(..., success: Boolean).  Call that from try and catch block with success = true and false.  This should get rid of the need of var noCatch?